tools/docker/python2.7/Dockerfile

```diff
@@ -59,10 +59,9 @@ ENV PYTHON_VERSION=2.7 \
 
 RUN mkdir ${TUTTLEOFX_BUILD} ${TRAVIS_BUILD_DIR}
 
-COPY . ${TUTTLEOFX_DEV}
+RUN git clone https://github.com/tuttleofx/TuttleOFX.git ${TUTTLEOFX_DEV}
```

Comment:
Ok so we need to find an other solution to get the source, because since the Dockerfile has been moved to "tools/docker/python2.7/", it is not possible to reference relative path out of the Docker wrokspace (ie. ../../..).